src/views/Ins.vue

```diff
@@ -1,103 +1,390 @@
+                <bar-chart :chart-data="sudanBarChartData" :options="barchartOption"></bar-chart>
+              </div>
+            </div>
+            <div class="country-wrapper">
+              <div class="text-container">
+                <input type="checkbox" id='kenya' v-bind:key="'Kenya'" :value="'Kenya'" v-model="checkedCountries">
+                <label for='kenya'><div></div>{{ kenyaLessons }} lessons </br> in Kenya</label>
+              </div>
+              <div class="chart-container">
+                <bar-chart :chart-data="kenyaBarChartData" :options="barchartOption"></bar-chart>
+              </div>
+            </div>
+            <div class="country-wrapper">
+              <div class="text-container">
+                <input type="checkbox" id='tanzania' v-bind:key="'Tanzania'" :value="'Tanzania'" v-model="checkedCountries">
+                <label for='tanzania'><div></div>{{ tanzaniaLessons }} lessons </br> in Tanzania</label>
+              </div>
+              <div class="chart-container">
+                <bar-chart :chart-data="tanzaniaBarChartData" :options="barchartOption"></bar-chart>
+              </div>
+            </div>
+            <div class="country-wrapper">
+              <div class="text-container">
+                <input type="checkbox" id='dr-congo' v-bind:key="'DR Congo'" :value="'DR Congo'" v-model="checkedCountries">
+                <label for='dr-congo'><div></div>{{ drcongoLessons }} lessons </br> in DR Congo</label>
+              </div>
+              <div class="chart-container">
+                <bar-chart :chart-data="drcongoBarChartData" :options="barchartOption"></bar-chart>
+              </div>
+            </div>
+              <!-- {{ summary }} -->
+          </column>
+      </row>
+    </section>
 
-  </div>
+  </body>
 </template>
 
 <script>
-  import LineChart from '../components/LineChart.js'
-  import {getCountries, getCamps, getSchools, getLessons} from '../data/data-provider.js'
+import LineChart from '../components/LineChart.js'
+import BarChart from '../components/BarChart.js'
+import { getCountries, getCamps, getSchools, getLessons } from '../data/data-provider.js'
 
-  export default {
-    components: {
-      LineChart
-    },
-    data () {
-      return {
-        selectedCountry: null,
-        selectedCamp: null,
-        selectedSchool: null,
-        chartData: {},
-        countries: [],
-        camps: [],
-        schools: []
+export default {
+  components: {
+    LineChart,
+    BarChart
+  },
+  data () {
+    return {
+      selectedCountry: null,
+      selectedCamp: null,
+      selectedSchool: null,
+      chartData: {},
+      sudanBarChartData: {},
+      kenyaBarChartData: {},
+      tanzaniaBarChartData: {},
+      drcongoBarChartData: {},
+      countries: [],
+      camps: [],
+      schools: [],
+      country: '',
+      school: '',
+      checkedCountries: [],
+      summary: '',
+      totalLessons: '',
+      sudanLessons: '',
```

Comment:
Hmm, could we somehow parametrize it by Country name, the hardcoding looks a bit ugly and will be hard to maintain